colossalai/shardformer/README.md

```diff
@@ -20,15 +20,20 @@
 The sample API usage is given below:
 
 ``` python
-from colossalai.shardformer import shard_model
+from colossalai.shardformer.shard import ShardConfig, shard_model
```

Comment:
We should make `ShardConfig` and `shard_model` the top-level API for the user in the `shardformer` module.

e.g. 

```python
from colossalai.shardformer import ShardConfig, shard_model
```